resources/views/custom_fields/fields/edit.blade.php

```diff
@@ -206,11 +217,13 @@
     $('#field_encrypted').on('ifChecked', function(event){
         $("#encrypt_warning").show();
         $("#show_in_email").hide();
+        $("#display_in_user_view").hide();
```

Comment:
I feel like we also need to enforce that these settings are forced to be set to zero when the field is encrypted.

My concern is - let's say you are creating a custom field, and you click 'show in email' and 'display in user view'. Then you click 'encrypt' - will it save 'show in email' as 1 still, even though the field is hidden? I'm not exactly sure.